templates/components/element_detail_tabs.html

```diff
@@ -292,7 +292,7 @@ <h4 id="panel-panel_num-title" class="panel-title">
                       </div>
                       <div class="form-group">
                         <input type="hidden" id="smt_id" name="smt_id" value="">
-                        <label for="statement">Statement</label>
+                        <label for="statement">What is the solution and how is it implemented?</label>
```

Comment:
Not sure why this text is changed. Would be confusing for the user when the accordion tabs and items have statement in the text. To consistent with the label and how we currently use it in the backend reverting this makes sense.

Reply:
Other pages actually read "What is the solution and how is it implemented?"

Reply:
If I was a user this would confuse me.